docs/usage/upstream-credentials.md

```diff
@@ -21,7 +21,7 @@ In order to pull private images through registry cache, it is required to supply
    EOF
    ```
 
-   For GCR, the username is `_json_key` and the password is the service account key in JSON format. To base64 encode the service account key, copy it and run:
+   For Artifact Registry, the username is `_json_key` and the password is the service account key in JSON format. To base64 encode the service account key, copy it and run:
```

Comment:
For Artifact Registry `_json_key_base64` is supported as well; ref https://cloud.google.com/artifact-registry/docs/docker/authentication#json-key

Reply:
Thanks for double-checking!